lte/gateway/release/build-magma.sh

```diff
@@ -326,6 +347,8 @@ $(glob_files "${SERVICE_DIR}/magma@mme.service" /etc/systemd/system/magma@mme.se
 $(glob_files "${SERVICE_DIR}/magma@sessiond.service" /etc/systemd/system/magma@sessiond.service) \
 $(glob_files "${SERVICE_DIR}/magma@mobilityd.service" /etc/systemd/system/magma@mobilityd.service) \
 $(glob_files "${SERVICE_DIR}/magma@pipelined.service" /etc/systemd/system/magma@pipelined.service) \
+$(glob_files "${SERVICE_DIR}/magma_dp@envoy.service" /etc/systemd/system/magma_dp@envoy.service) \
```

Comment:
This will automate the installation of the service, nice. that's why you remove `name: Copy envoy service file`